### DISCUSSION TITLE
Elasticsearch diver

### DISCUSSION THREAD

**Discussion by rajmundtoth0** [owen-it/laravel-auditing#972] [Ideas]
Hi,

First, thank you for this package! We use this on a daily basis.

I have noticed that the community Elasticsearch driver is quite outdated and is not compatible with Elasticsearch 8.

I made a new one and I would highly appreciate if you could take a look and maybe include it in here.

https://github.com/rajmundtoth0/laravel-auditing-elasticsearch-driver

Thank you and all the best!

Rajmund Toth

**Comment by willpower232** [comment]
If you make a PR over at https://github.com/owen-it/laravel-auditing.com then someone can double check and update the site at the same time.
